bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,109 @@
+# Stellar accounts as a service
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 2000 XLM | 800 XLM | Continuous
```

Comment:
I think this should be more around 1000 - 1500 for the reviewer and maybe 500 for the reviewer.

Reply:
I based it on the the rewards for the [basic account viewr](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-1/basic-account-viewer.md). I think that this bounty is much harder.